src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -73,6 +73,27 @@ export default class ComparisonGroupManager extends React.Component<IComparisonG
         this.addSamplesTargetGroupId = "";
     }
 
+    @autobind
+    @action
+    private deleteGroup(group: StudyViewComparisonGroup) {
+        if (this.props.store.isComparisonGroupSelected(group.uid)) {
```

Comment:
we no longer need to check if it's selected. we're going to mark it for deletion regardless, because we're clicking directly on it to delete it.